sys-cluster/reframe/reframe-3.9.1.ebuild

```diff
@@ -24,11 +33,12 @@ RDEPEND="dev-python/argcomplete[${PYTHON_USEDEP}]
 	dev-python/semver[${PYTHON_USEDEP}]
 	>=sys-apps/archspec-0.1.2"
 BDEPEND="
-	dev-python/pyproject2setuppy[${PYTHON_USEDEP}]
 	test? (
 		>=dev-python/pytest-6.2.3[${PYTHON_USEDEP}]
 		>=dev-python/pytest-forked-1.3.0[${PYTHON_USEDEP}]
 	)"
+# removed
+#	dev-python/pyproject2setuppy[${PYTHON_USEDEP}]
```

Comment:
unless we add a more meaningful comment, just remove these lines?

Reply:
Removed.